### ISSUE TITLE
ms-swift-3.4.0.dev0 推理seq_cls(文本序列分类)未能输出logprobs

### ISSUE THREAD

**Issue by regretBigShark** [modelscope/ms-swift#3976]
我使用Qwen2.5-0.5B-Instruct作为base_model，在自定数据集上微调做文本分类（共20类）。微调代码如下：
`CUDA_VISIBLE_DEVICES=0,1 \
swift sft \
    --model Qwen/Qwen2.5-0.5B-Instruct \
    --train_type lora \
    --dataset ./my-dataset/20news/train_20news.json \
    --split_dataset_ratio 0.2 \
    --torch_dtype bfloat16 \
    --num_train_epochs 5 \
    --per_device_train_batch_size 8 \
    --per_device_eval_batch_size 8 \
    --learning_rate 1e-4 \
    --lora_rank 8 \
    --lora_alpha 32 \
    --target_modules all-linear \
    --gradient_accumulation_steps 16 \
    --eval_steps 50 \
    --save_steps 50 \
    --save_total_limit 2 \
    --logging_steps 5 \
    --max_length 8192 \
    --output_dir output-20news-logits \
    --warmup_ratio 0.05 \
    --dataloader_num_workers 4 \
    --num_labels 20 \
    --task_type seq_cls \
    --use_chat_template true \
    --system "You are a helpful assistant for news classification." \
    --logprobs true \
    --top_logprobs 3 `

在微调完成后，推理代码如下：
`CUDA_VISIBLE_DEVICES=1 \
swift infer \
    --adapters output-20news-logits/v1-20250422-160647/checkpoint-1000 \
    --infer_backend pt \
    --temperature 0 \
    --max_new_tokens 4096 \
    --logprobs true \
    --top_logprobs 3 \
    --load_data_args false \
    --stream false`

在终端输入待测文本后，输出仅有一个label，没有logprobs。查看推理保存的结果，仍然也没有logprobs。

请问我应该如何修改，使得推理时我能够获得top3的label以及相应的logprobs？

**Comment by Jintao-Huang** [comment]
试试增加`--val_dataset <...>`
